layersvt/screenshot.cpp

```diff
@@ -678,9 +678,16 @@ static void writePPM(const char *filename, VkImage image1) {
 
     // Set up the command buffer.  We get a command buffer from a pool we saved
     // in a hooked function, which would be the application's pool.
+    if (deviceMap[device]->commandPools.empty()) {
+        assert(!deviceMap[device]->commandPools.empty());
```

Comment:
I put the assert in because all the previous errors in this function which return, also have an assert. I agree they will have different behavior in release and debug, but was just following the existing behavior.